src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativeNonparametricStabilityJudge.kt

```diff
@@ -4,11 +4,11 @@ import com.atlassian.performance.tools.jiraactions.api.ActionType
 import com.atlassian.performance.tools.report.ActionMetricsReader
 import com.atlassian.performance.tools.report.api.ShiftedDistributionRegressionTest
 import com.atlassian.performance.tools.report.api.junit.FailedAssertionJUnitReport
-import com.atlassian.performance.tools.report.api.junit.JUnitReport
 import com.atlassian.performance.tools.report.api.junit.SuccessfulJUnitReport
 import com.atlassian.performance.tools.report.api.result.EdibleResult
 import com.atlassian.performance.tools.report.junit.FailedActionJunitReport
 
+@Deprecated("This class will no longer be a part of API, as the results produced by this judge may be misleading.")
```

Comment:
Could we be more precise on what does "misleading" mean?

Reply:
Nice-to-have: test showing this overly sensitive verdict. Bonus: the same test data can be used to develop better alternatives.